extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -328,7 +328,7 @@ public function loadDataFromRevDataProvider() {
 				'parent_page_id' => '2147',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "parent_page_id" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)